torch/headeronly/macros/Export.h

```diff
@@ -130,22 +130,14 @@
 #define TORCH_CUDA_CU_API C10_IMPORT
 #endif
 
-#if defined(TORCH_HIP_BUILD_MAIN_LIB)
-#define TORCH_HIP_CPP_API C10_EXPORT
-#define TORCH_HIP_API C10_EXPORT
-#else
-#define TORCH_HIP_CPP_API C10_IMPORT
-#define TORCH_HIP_API C10_IMPORT
-#endif
-
 #if defined(TORCH_XPU_BUILD_MAIN_LIB)
 #define TORCH_XPU_API C10_EXPORT
 #else
 #define TORCH_XPU_API C10_IMPORT
 #endif
 
 // Enums only need to be exported on windows for non-CUDA files
-#if defined(_WIN32) && defined(__CUDACC__)
+#if defined(_WIN32) && defined(__HIPCC__)
```

Comment:
@pragupta The changes to the comments in this file and the change in this line seem to contradict the PR description: "macros/Export.h: remove TORCH_HIP_CPP_API/TORCH_HIP_API as CUDA ones get hipified and converted correctly (need to upstream this)"?